README.md

```diff
@@ -76,6 +77,29 @@ Finally, create a home page in the CMS.
 
 *NOTE*: TACC VPN or physical connection to the TACC network is required To log-in to CMS using LDAP, otherwise the password set with `python3 manage.py createsuperuser` is used
 
+### Setting up search index and initial project id locally:
+
+At least one page in CMS is needed (see above), then rebuild the cms search index
+```
+    docker exec -it core_portal_cms /bin/bash
+    python3 manage.py rebuild_index
+```
+Then use the django shell in the `core_portal_django` container:
+```
+    docker exec -it core_portal_django /bin/bash
+    python3 manage.py shell
+```
+to run the following code to set up the projects index and initial project id
+```
+from portal.libs.elasticsearch.indexes import setup_files_index, setup_projects_index, setup_allocations_index
+setup_files_index()
```

Comment:
@jarosenb is `force=True` as an option worth noting here?

Reply:
Yeah let's use `force=True` as the default, it will stop a lot of common search bugs.

Reply:
👍  cool updated!